src/components/pages/order/main/MenuCard.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react'
+import styled from 'styled-components';
+import { fakeMenu2 } from '../../../../fakeData/fakeMenu';
+import { theme } from '../../../../theme';
+import Card from '../../reusable-ui/Card';
+
+export default function MenuCard() {
```

Comment:
```suggestion
export default function Menu() {
```

MenuCard pourrait faire croire à une Card dans le Menu 😕

Reply:
changement de naming fait